### PR TITLE
369: More point refactoring

### PR DESCRIPTION
**Pull request by pimotte**
ポケモンを捕まえよう！ = Let's catch Pokémon!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -447,17 +447,16 @@ condition on said PR, for 0.1 points.
 Players are encouraged to present a quote that matches the contents of the pull request that has not been used before in
 another Pull Request.
 
-**308** *Hash Brownie Points*
+**369** *Hash Brownie Points*
 
-If a PR that outlines a rule-change contains `y` commit hashes containing an English dictionary word of at least
-four consecutive letters, the proposer will be awarded `y` additional points when the PR is merged into the master
-branch of pimotte/nomic.
+The presense of  `y` commit hashes containing an English dictionary word of at least
```
Comment (MrHug) [inline README.md]: What are we sensing beforehand?

**Comment by MrHug** [comment]
IF the typo is fixed THEN :+1:

**Comment by pimotte** [comment]
Typo fixed, vote not invalidated:D

Edit: Wth happened to fixed?

**Comment by Jason-Statham** [comment]
pimotte edited a message. The previous content of the message was:
Typo fixed, vote not invalidated:D

**Comment by pimotte** [comment]
I'm issuing a warning on this.

**Comment by pimotte** [comment]
2/2, merging this.

Edit: With the note that the typo has been fixed, so @MrHug s vote resolves to :+1:

**Comment by Jason-Statham** [comment]
pimotte edited a message. The previous content of the message was:
2/2, merging this.

**Comment by MrHug** [comment]
Well if you have to... :wink:

**Pull request merged**